python-threatexchange/threatexchange/cli/experimental_fetch.py

```diff
@@ -23,9 +25,15 @@ class ExperimentalFetchCommand(command_base.Command):
     """
 
     PROGRESS_PRINT_INTERVAL_SEC = 30
+    DEFAULT_REFETCH_SEC = 3600 * 24 * 85  # 85 days
```

Comment:
(blocking): I think this is fundamentally incorrect. Your tombstones only last for 30 days, right? At 31 days you'll no longer be able to see the delete update that would put the dataset in the right state.

Reply:
Currently tombstones last for 90 days

Reply:
Chatted with offline, the current impl is indeed 90 days. Suggest putting a comment here about why refetch is needed in general (and it's relation to the tombstone duration.

Reply:
Added comment